build-tools/src/main/resources/software/amazon/awssdk/spotbugs-suppressions.xml

```diff
@@ -197,6 +197,12 @@
         <Bug pattern="RCN_REDUNDANT_NULLCHECK_OF_NONNULL_VALUE"/>
     </Match>
 
+    <!-- See SpotBugs bug: https://github.com/spotbugs/spotbugs/issues/600, https://github.com/spotbugs/spotbugs/issues/756 -->
+    <Match>
+        <Class name="software.amazon.awssdk.core.SdkServiceClientConfiguration$Builder"/>
```

Comment:
SpotBugs gets confused about the nullability of the `overrideConfiguration`, probably because is looking at the default method that throws an `UnsupportedOperationException`. Adding this since I'm certain that this value **can** be null.